.github/workflows/ci.yml

```diff
@@ -92,7 +93,7 @@ jobs:
         cd build.cmake
         cmake --version
         cmake -DCMAKE_CXX_COMPILER=${{ matrix.ci.compiler }} -DCMAKE_CXX_STANDARD=${{ matrix.ci.cxxstd }} -DBUILD_DOCUMENTATION=YES -DCMAKE_BUILD_TYPE=${{ matrix.ci.build_type }} ..
-        cmake --build . -j 2
+        make -j 2
```

Comment:
What's this change for?

Reply:
with cmake version 3.10.2
```
$ cmake --build . -j 2
Unknown argument -j
Unknown argument 2
Usage: cmake --build <dir> [options] [-- [native-options]]
Options:
  <dir>          = Project binary directory to be built.
  --target <tgt> = Build <tgt> instead of default targets.
                   May only be specified once.
  --config <cfg> = For multi-configuration tools, choose <cfg>.
  --clean-first  = Build target 'clean' first, then build.
                   (To clean only, use --target 'clean'.)
  --use-stderr   = Ignored.  Behavior is default in CMake >= 3.0.
  --             = Pass remaining options to the native tool.
```
we could use `-- -j 2`, if that's preferred?

Reply:
Thanks, no need to change. I didn't realize that flag was such a recent addition.

Reply:
It's a [3.12 feature](https://cmake.org/cmake/help/latest/release/3.12.html), to be specific.